src/data/knowledge_graph.csv

```diff
@@ -12786,42 +12786,775 @@ IL_Z_9200,Q152368,
 IM,Q9676,country/IMN
 IN,Q668,country/IND
 IN_AN,Q40888,
+IN_AN_NIC,Q797295
+IN_AN_NMA,Q796983
+IN_AN_SAS,Q796979
 IN_AP,Q1159,
+IN_AP_ANT,Q15212
+IN_AP_CHI,Q15213
+IN_AP_CUD,Q15342
+IN_AP_EGI,Q15338
+IN_AP_GNT,Q15341
+IN_AP_KRS,Q15382
+IN_AP_KUR,Q15381
+IN_AP_NEL,Q15383
+IN_AP_PRA,Q15390
+IN_AP_SRK,Q15395
+IN_AP_VNG,Q15392
+IN_AP_VSK,Q15394
+IN_AP_WGI,Q15404
 IN_AR,Q1162,
+IN_AR_ANJ,Q15413
+IN_AR_CHG,Q15427
+IN_AR_DBN,Q15446
+IN_AR_EKG,Q15424
+IN_AR_ESG,Q15419
+IN_AR_KKY,Q2449506
+IN_AR_KML,Q48731073
+IN_AR_KRD,Q21018627
+IN_AR_LDG,Q5627568
+IN_AR_LDV,Q2373368
+IN_AR_LHT,Q15438
+IN_AR_LPR,Q63563632
+IN_AR_LSI,Q15436
+IN_AR_LWR,Q13602925
+IN_AR_NMS,Q21559824
+IN_AR_PKS,nan
```

Comment:
```suggestion
IN_AR_PKS,
```

Reply:
thanks, good catch! Yeah there were a few (2) unresolved districts, looks like I forgot to get rid of one nan.